script_01.py

```diff
@@ -0,0 +1,71 @@
+import math
+import cmath
+
+def task1(text="Count stuff in this random sentence."):
+    print(text)
+    vowels=['a','e','i','o','u','y']
+    consonants=list('bcdfghjklmnpqrstvwxyz')
+    sumofv=0
```

Comment:
variable names should be more explanatory, e.q. `sum_of_vovels` instead of `sumofv`